### ISSUE TITLE
Unknown package format when reimporting a package with same name as previously imported

### ISSUE THREAD

**Issue by riesaau** [chamilo/chamilo-lms#4798]
Chamilo 1.11.20

If I import a scorm package into a learning path, then I delete it, then i import it again one or two times more.  Eventually it will no longer import, giving the above message "unknown package format"

1. Under learning paths (for my course), go to the middle icon which is to import a scorm pacakge, and browse to zip file on my computer of published scorm package
2. Import the zip
3. delete the module using the red X
4. Import it again and delete it (may have to repeat these steps one more time)
5. Attempt to import the zip again
6. Get the error message "unknown package format"
7. Change the name of the zip on my computer
8. Reimport it again
9. Imports successfully

**Expected behavior**
I expected I should be able to import the course as many times as I want after I delete it, without needing to change the name of the zip file I'm importing.

**Screenshots**

![Screenshot_2023-07-09_at_00_29_23](https://github.com/chamilo/chamilo-lms/assets/97856892/e6b0e6c3-4f4d-4b21-a573-66caf97a472e)

**Comment by riesaau** [comment]
I changed the name of the zip file i was importing slightly and now it imports again. 
![Screenshot 2023-07-09 at 00 38 35](https://github.com/chamilo/chamilo-lms/assets/97856892/c2a14238-7c9f-4527-8977-cd3dd2d18950)

**Comment by ywarnier** [comment]
Yes, this is a bug known by a few of us. Hopefully we can work on this for future versions (it might be fixed through other changes anyway). Thanks for reporting.

**Comment by ywarnier** [comment]
This might have been fixed in v3 but we're lacking time to confirm it, so I'm moving this to v4
